<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncoderWidget</class>
 <widget class="QWidget" name="EncoderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encoder</string>
  </property>
  <layout class="QHBoxLayout" name="central_layout">
   <item>
    <widget class="QPlainTextEdit" name="textedit_original">
     <property name="placeholderText">
      <string>原文在这里切噜~</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_control">
     <item>
      <widget class="QPushButton" name="button_from_string">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_to_string">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textedit_translated">
     <property name="placeholderText">
      <string>切噜～♪</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
